### ISSUE TITLE
Creating SearchView is failure.

### ISSUE THREAD

**Issue by trancebeat** [JakeWharton/ActionBarSherlock#781]
I'm using ActionBarSherlock 4.2.0.

On Android 2.3 or older, a creating SearchView instance shows the following error message.

```
Could not find class 'com.actionbarsherlock.widget.SearchView$11', referenced from method com.actionbarsherlock.widget.SearchView.<init>
```

But, this message is not shown on Android 4.1 device.

Since `android.view.View.OnLayoutChangeListener` is available on Android 3.0 or later, the above problem is occurred.

SearchView.java, line 338:

```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
    mDropDownAnchor.addOnLayoutChangeListener(new OnLayoutChangeListener() {
        @Override
        public void onLayoutChange(View v, int left, int top, int right, int bottom,
                                                             int oldLeft, int oldTop, int oldRight, int oldBottom) {
            adjustDropDownSizeAndPosition();
        }
    });
} else {
```

In this case, I think that ClassLoader or lazy class loading should be used.

**Comment by JakeWharton** [comment]
Does it crash? The classloader should just evict the class since it's type isn't present and we never call it so there's no runtime crash.

**Comment by trancebeat** [comment]
No crash in runtime.

But, the Eclipse debugger is stopped when I have debugged on Android 2.3 device, and the logcat shows this error message.
When my app is running on Android 4.1 device, the debugger does not stop and show no error logcat message.

I found like this issue in stackoverflow.

http://stackoverflow.com/questions/14110399/classnotfoundexception-with-actionbarsherlock-searchview

I only feel strange about this issue.
